Component/Pet/Pet.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+const StyledPet = styled.div`
+  display: flex;
+  border: 2px solid #000000;
+  border-radius: 10px;
+  margin: 10px;
+  padding: 10px;
+  justify-content: space-around;
+  align-items: center;
+  max-width: 250;
+  background-color: var(--secondary-color);
+  span {
+    font-size: 20vw;
+    position: absolute;
+  }
+  .portrait {
```

Comment:
Nested styles are generally not recommended because they can make the code harder to maintain and read. They create tightly coupled styles that are less reusable across different components, making it difficult to manage and scale the CSS. 

I suggest creating a styled component for your `span` and portrait.
Additionally, please use only styled components and do not mix them with CSS classes.